client/post-editor/editor-mobile-navigation/index.jsx

```diff
@@ -23,17 +53,40 @@ const EditorMobileNavigation = React.createClass( {
 
 		return (
 			<div className="editor-mobile-navigation">
-				<Site indicator={ false } site={ this.props.site } />
```

Comment:
Could we remove the `this.props.site` condition above now? Seems odd that we aren't able to render the mobile action bar 'til sites are loaded. I followed where it's used in `<EditorPublishButton />` and it looks that it's used by `siteUtils.userCan` which gracefully handles lack of a site ([source](https://github.com/Automattic/wp-calypso/blob/0cfd9a6296bdbacf94bed4ec49d805c445c81c56/client/lib/site/utils.js#L7-L9)).